server/src/main/java/org/elasticsearch/cluster/routing/ShardRouting.java

```diff
@@ -615,20 +623,20 @@ public String shortSummary() {
     @Override
     public XContentBuilder toXContent(XContentBuilder builder, Params params) throws IOException {
         builder.startObject()
-            .field("state", state())
-            .field("primary", primary())
-            .field("node", currentNodeId())
-            .field("relocating_node", relocatingNodeId())
-            .field("shard", id())
-            .field("index", getIndexName());
+            .field(Fields.STATE, state())
```

Comment:
We've stopped making these `Fields` objects in a past year or so and just started to use string constants or even quoted strings.